assets/javascripts/discourse/templates/admin-plugins-gamification.hbs

```diff
@@ -1,49 +1,67 @@
 {{#if selectedLeaderboard}}
   {{d-button
-    class="leaderboard-back"
+    class="leaderboard-admin__btn-back btn-flat"
     icon="chevron-left"
     label="gamification.back"
     title="gamification.back"
     action=(action (mut selectedLeaderboardId) null)
   }}
   <form class="form-vertical">
     <div class="control-group">
-      <label class="control-label">{{i18n "gamification.leaderboard.name"}}</label>
-      {{input
-        type="text"
-        value=selectedLeaderboard.name
-        placeholder=(i18n "gamification.leaderboard.name")
-      }}
-      <label class="control-label">{{i18n "gamification.leaderboard.date.range"}}</label>
-      <div class="controls">
-        {{date-picker-past
-          class="date-input"
-          value=selectedLeaderboard.from_date
-          onSelect=(action (mut fromDate))
+      <div class="leaderboard-admin__form-row">
```

Comment:
This is part of a larger discussion, but I want to highlight an example of a consistency problem here. Core uses `control-group` for groupings of form elements. I know that core isn't super consistent across the different screens, that is something we should certainly try and fix. But that side, having a new class name in a plugin introduces even more inconsistency, any theme that styles `control-group`s won't be applying those styles to this particular grouping of form elements (and same for any future core stylesheet changes).

Reply:
Ah ok so it has to be ```control-group``` then?